WordPressAuthenticator/Unified Auth/View Related/VerifyEmail/VerifyEmailViewController.swift

```diff
@@ -0,0 +1,224 @@
+import UIKit
+
+final class VerifyEmailViewController: LoginViewController {
+
+    // MARK: Properties
+
+    @IBOutlet private weak var tableView: UITableView!
+    private let rows = Row.allCases
+
+    // MARK: - Actions
+    @IBAction private func handleSendLinkButtonTapped(_ sender: NUXButton) {
+        configureViewLoading(false)
+        tracker.track(click: .requestMagicLink)
+        requestAuthenticationLink()
+    }
+
+    // MARK: - View lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        navigationItem.title = WordPressAuthenticator.shared.displayStrings.logInTitle
+        styleNavigationBar(forUnified: true)
+
+        // Store default margin, and size table for the view.
+        defaultTableViewMargin = tableViewLeadingConstraint?.constant ?? 0
+        setTableViewMargins(forWidth: view.frame.width)
+
+        localizePrimaryButton()
+        registerTableViewCells()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        if isBeingPresentedInAnyWay {
+            tracker.track(step: .verifyEmailInstructions)
+        } else {
+            tracker.set(step: .verifyEmailInstructions)
+        }
+    }
+
+    // MARK: - Overrides
+
+    override var sourceTag: WordPressSupportSourceTag {
+        .verifyEmailInstructions
```

Comment:
https://github.com/woocommerce/woocommerce-ios/blob/trunk/WooCommerce/Classes/Authentication/AuthenticationManager.swift#L385

I hope that Zendesk uses the `sourceTag` to  identify the screen from which the support request was initiated from. Since, I am adding a new screen I decided to add a `sourceTag` as well.